README.md

```diff
@@ -31,507 +30,233 @@ scp -i benchmarks.pem -r remote-setup $SSH_ADDRESS:
-         /fortunes      3.6      orm        -       32       3,466     2,004 |-42%     3,351 | -3% 
-         /fortunes      3.6      orm        -       64       3,927     3,781 | -4%     4,519 | 15% 
-         /fortunes      3.6      orm        -      128       3,863     3,879 |  0%     4,606 | 19% 
-         /fortunes      3.6      orm        -      256       4,036     4,042 |  0%     4,486 | 11% 
-         /fortunes      3.6      raw        -       32       4,930     4,602 | -7%     5,931 | 20% 
-         /fortunes      3.6      raw        -       64       4,984     4,927 | -1%     6,194 | 24% 
-         /fortunes      3.6      raw        -      128       5,263     5,198 | -1%     6,386 | 21% 
-         /fortunes      3.6      raw        -      256       5,423     5,346 | -1%     6,553 | 21% 
-
-             /json      3.6        -        -       32      11,933     6,674 |-44%    22,730 | 90% 
-             /json      3.6        -        -       64      14,983    14,379 | -4%    26,269 | 75% 
-             /json      3.6        -        -      128      15,759    14,554 | -8%    25,988 | 65% 
-             /json      3.6        -        -      256      15,933    14,352 |-10%    25,778 | 62% 
-
-        /plaintext      3.6        -        -       32      12,334    13,836 | 12%    24,245 | 97% 
-        /plaintext      3.6        -        -       64      14,767    14,098 | -5%    24,506 | 66% 
-        /plaintext      3.6        -        -      128      15,293    13,669 |-11%    23,282 | 52% 
-        /plaintext      3.6        -        -      256      14,769    13,812 | -6%    24,649 | 67% 
-
-/queries/{queries}      3.6      orm        5      256       1,638     1,606 | -2%     1,684 |  3% 
-/queries/{queries}      3.6      orm       10      256         827       881 |  7%       965 | 17% 
-/queries/{queries}      3.6      orm       20      256         436       464 |  6%       486 | 11% 
-/queries/{queries}      3.6      raw        5      256       4,503     4,437 | -1%     5,639 | 25% 
-/queries/{queries}      3.6      raw       10      256       3,229     3,091 | -4%     3,571 | 11% 
-/queries/{queries}      3.6      raw       20      256       2,009     2,022 |  1%     2,252 | 12% 
-
-/updates/{queries}      3.6      orm        5      256         829       830 |  0%       858 |  3% 
-/updates/{queries}      3.6      orm       10      256         405       451 | 11%       450 | 11% 
-/updates/{queries}      3.6      orm       20      256         223       229 |  3%       198 |-11% 
-/updates/{queries}      3.6      raw        5      256       2,800     2,763 | -1%     3,076 | 10% 
-/updates/{queries}      3.6      raw       10      256       1,744     1,724 | -1%     1,869 |  7% 
-/updates/{queries}      3.6      raw       20      256         959     1,000 |  4%     1,053 | 10%
+               URL   python       DB  queries     Conc        3.6             3.7
+             /json      3.7        -        -       32      7,140     6,723 | -6%
```

Comment:
If I understand this correctly, these numbers are the number of requests per second or similar? Therefore higher number is better? Can we add this information just before the table, so it's clearer what we're looking at?

Reply:
Good point, fixed!